src/dsp_tools/utils/project_create.py

```diff
@@ -35,51 +39,51 @@ def _create_project_on_server(
         verbose: verbose switch
 
     Raises:
-        BaseError: if the project cannot be created on the DSP server
+        UserError: if the project cannot be created on the DSP server
 
     Returns:
         a tuple of the remote project and the success status (True if everything went smoothly, False otherwise)
     """
     try:
         # the normal, expected case is that this try block fails
         project_local = Project(con=con, shortcode=project_definition["project"]["shortcode"])
-        project_remote: Project = try_network_action(
-            action=lambda: project_local.read(),
-            failure_msg=""
-        )
-        print(f"\tWARNING: Project '{project_remote.shortname}' ({project_remote.shortcode}) already exists on the DSP "
-              f"server. Updating it...")
-        success = False
+        project_remote: Project = try_network_action(lambda: project_local.read())
```

Comment:
```suggestion
        project_remote: Project = try_network_action(project_local.read)
```
This is beside the topic, but just because it caught my eye: I'm not 100% confident, but I think you don't need the lambda here, if you just pass the method as a function

Reply:
When I introduced this, I made experiments and I found out that the lambda is necessary. It also makes sense from a theoretical point of view: 
`try_network_action(project_local.read)` passes **the result** of `read()` to the method, which is exactly what we **don't** want.
On the other hand, `try_network_action(lambda: project_local.read())` passes the `read()` method itself to the method, and that's what we want

Reply:
no, as far as I know, the difference between  
`try_network_action(project_local.read)` and   
`try_network_action(project_local.read())`  
should be that one passes the function and the other the evaluation of the function - and the lambda is also just a function pointing to `project_local.read`.

Reply:
Ah, now I see, you have omitted the `()` in your first comment. Sorry, I wasn't attentive enough.

That's actually a good point. You're probably right and I could try that out. Now, I already merged, but I will keep it in mind.